README.md

```diff
@@ -31,140 +31,192 @@ This repository hosts an e-commerce project tailored for baby products. It is de
 ### Steps
 
 1. **Clone the repository:**
-   ```bash
-   git clone https://github.com/your-repo-name
-   ```
+
+```bash
+# Bash
+git clone https://github.com/your-repo-name
+```
+
 2. **Set up the virtual environment:**
-   ```bash
-   cd {project_directory}
-   python -m venv env
-   source env/bin/activate  # On Windows: `env\Scripts\activate`
-   ```
+
+```bash
+# Bash
+cd {project_directory}
+python -m venv env
+```
+
+**On Windows:**
+
+```cmd
+# CMD
+env\Scripts\activate
+```
+
+**On MacOS/Linux:**
+
+```bash
+# Bash
+source env/bin/activate
+```
+
 3. **Install dependencies:**
-   ```bash
-   pip install -r requirements.txt
-   ```
+
+```bash
+# Bash
+pip install -r requirements.txt
+```
+
 4. **Set up environment variables:**
-   Create a `.env` file with the following values:
-   ```env
-   SECRET_KEY={your_secret_key}
-   DEBUG=True
-   ALLOWED_HOSTS=localhost,127.0.0.1
-   ```
-5. **Run database migrations:**
-   ```bash
-   python manage.py migrate
-   ```
+   Create a .env file with the following values:
+
+```env
+SECRET_KEY={your_secret_key}
+DEBUG=True
+ALLOWED_HOSTS=localhost,127.0.0.1
+```
+
+5. **Make and run database migrations:**
+
+```bash
+# Bash
+python manage.py makemigrations
+python manage.py migrate
+```
+
 6. **Start the development server:**
-   ```bash
-   python manage.py runserver
-   ```
-7. **Access the application:** Open your browser and navigate to `http://127.0.0.1:8000/`.
+
+```bash
+# Bash
+python manage.py runserver
+```
+
+7. **Access the application:** Open your browser and navigate to http://127.0.0.1:8000/.
 
 ## Usage
 
 ### Configuration Details
 
-- **Django Settings:** Located in `babyshop_app/babyshop/settings.py`. This file allows you to modify core configurations, such as database settings, static files, and application-specific settings.
-- **Routing:** URL configurations are managed in `urls.py` files across subdirectories. Customize these files to add or modify routes as needed.
-- **Environment Variables:** Use a `.env` file to securely configure sensitive data. Ensure the `.env` file is excluded from version control by adding it to `.gitignore`.
+- **Django Settings:** Located in babyshop_app/babyshop/settings.py. This file allows you to modify core configurations, such as database settings, static files, and application-specific settings.
```

Comment:
You can point to files in your repository using this Markdown syntax: [Your File Name](./path/to/file)

Located [here](./babyshop/settings.py).